README.md

```diff
@@ -58,6 +64,13 @@ Reference contracts that the Helix system uses to make everything trustless.
 Everything comes together here. This is a simulated environment where you can see how to wire
 everything up and even tweak with some parameters for your dApp.
 
-## License
+### [`notebooks`](packages/examples/notebooks/)
+
+Early simulations of the pay-it-backward model.
+
+## Some other notable items...

```

Comment:
We should update the table of contents. Markdownify used the word `Related` in it's. https://github.com/amitmerchant1990/electron-markdownify/blob/master/README.md#related

Reply:
Sorry, I'm not sure I follow. Where does the front-matter in HTML in the README.md file come from?